src/v0/destinations/sendgrid/deleteUsers.js

```diff
@@ -1,17 +1,41 @@
-const _ = require("lodash");
 const { httpDELETE } = require("../../../adapters/network");
 const ErrorBuilder = require("../../util/error");
-const {
-  MAX_BATCH_SIZE,
-  DESTINATION,
-  DELETE_CONTACTS_ENDPOINT
-} = require("./config");
+const { DESTINATION, DELETE_CONTACTS_ENDPOINT, urlLimit } = require("./config");
```

Comment:
Can we update the name of `urlLimit` to `DELETE_MAX_BATCH_SIZE ` and add documentation if availale?

Reply:
I have used the urlLimit to be clear with the approach of what we are doing and this limit is found based on the trial and error method. So no doc is available.